backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/project/workorder/export/workOrderProjectExport.feature

```diff
@@ -0,0 +1,100 @@
+Feature: Validate work order project is sent to RabbitMQ
```

Comment:
PLease give more love to this feature name. what does "validate" mean? and it's not tje project that is send to rabbitmq, right?